Libraries/plist/CMakeLists.txt

```diff
@@ -69,6 +69,11 @@ add_executable(plutil Tools/plutil.cpp)
 target_link_libraries(plutil plist process util)
 install(TARGETS plutil DESTINATION usr/bin)
 
+add_executable(PlistBuddy Tools/PlistBuddy.cpp ${CMAKE_SOURCE_DIR}/ThirdParty/linenoise/linenoise.c)
+target_include_directories(PlistBuddy PRIVATE "${CMAKE_SOURCE_DIR}/ThirdParty/linenoise")
+target_link_libraries(PlistBuddy plist util)
+install(TARGETS PlistBuddy DESTINATION usr/bin)
```

Comment:
This is the first thing that requires `git submodule init` to build — @strager, is that still an issue?